### PR TITLE
Added flags for postgres forwarding and Docker commands

### PR DESCRIPTION
**Pull request by potiuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,14 +124,30 @@ Last used workspace, project, key and python version are used in this case:
...
+If you want to forward a port for using the webserver, use the --forward-webserver-port flag:
 
 ```
-./run_environment.sh --forward-port 8080
+./run_environment.sh --forward-webserver-port 8080
 ```
 
+Note that you need to start the webserver manually with `airflow webserver` command.
+
+## Forwarding postgres port to Airflow's Postgres DB
```
Comment (sprzedwojski) [inline README.md]:
```suggestion
## Forwarding Postgres port to Airflow's Postgres DB
```

File: README.md [hunk 2]
```diff
@@ -124,14 +124,30 @@ Last used workspace, project, key and python version are used in this case:
...
+
+If you want to forward a port for using the webserver, use the --forward-postgres-port flag:
```
Comment (sprzedwojski) [inline README.md]:
```suggestion
If you want to forward a port for remotely accessing the database, use the --forward-postgres-port flag:
```

File: README.md [hunk 3]
```diff
@@ -257,10 +283,15 @@ Flags:
...
         Python virtualenv used by default. One of ('2.7', '3.5', '3.6'). [2.7]
 
--f, --forward-port <PORT_NUMBER>
+-f, --forward-webserver-port <PORT_NUMBER>
         Optional - forward the port PORT_NUMBER to airflow's webserver (you must start
         the server with 'airflow webserver' command manually).
 
+-F, --forward-postgres-port <PORT_NUMBER>
+        Optional - forward the port PORT_NUMBER to airflow's Postgres database. You can
+        login to the database as root user with airflow password. Database of airflow
```
Comment (sprzedwojski) [inline README.md]:
```suggestion
        login to the database as the user "root" with password "airflow". Database of airflow
```

File: README.md [hunk 4]
```diff
@@ -257,10 +283,15 @@ Flags:
...
+        is named airflow/airflow.db
```
Comment (sprzedwojski) [inline README.md]:
```suggestion
        is named "airflow/airflow.db".
```

File: run_environment.sh [hunk 5]
```diff
@@ -247,10 +247,15 @@ Flags:
...
         Python virtualenv used by default. One of ('2.7', '3.5', '3.6'). [2.7]
 
--f, --forward-port <PORT_NUMBER>
+-f, --forward-webserver-port <PORT_NUMBER>
         Optional - forward the port PORT_NUMBER to airflow's webserver (you must start
         the server with 'airflow webserver' command manually).
 
+-F, --forward-postgres-port <PORT_NUMBER>
+        Optional - forward the port PORT_NUMBER to airflow's Postgres database. You can
+        login to the database as root user with airflow password. Database of airflow
```
Comment (sprzedwojski) [inline run_environment.sh]:
```suggestion
        login to the database as the user "root" with password "airflow". Database of airflow
```

File: run_environment.sh [hunk 6]
```diff
@@ -247,10 +247,15 @@ Flags:
...
+        is named airflow/airflow.db
```
Comment (sprzedwojski) [inline run_environment.sh]:
```suggestion
        is named "airflow/airflow.db".
```

**Review by sprzedwojski — CHANGES_REQUESTED** [review]

**Comment by potiuk** [comment]
@sprzedwojski - thanks for corrections. It should build now much faster and we can merge it so that we can merge Antek's configuration change.

**Comment by potiuk** [comment]
@sprzedwojski  -> it builds now :)

**Review by sprzedwojski — APPROVED** [review]

**Pull request merged**